### PR TITLE
chore: Case types

### PR DESCRIPTION
**Pull request by andytaylorkainos**

### CODE REVIEW

File: conf/constants.py [hunk 1]
```diff
@@ -49,10 +49,10 @@
 DOWNLOAD_URL = "/download/"
 
 # Applications constants
-STANDARD_LICENCE = "standard_licence"
-OPEN_LICENCE = "open_licence"
-HMRC_QUERY = "hmrc_query"
-EXHIBITION_CLEARANCE = "exhibition_clearance"
+STANDARD = "standard"
```
Comment (Mark-Spiers-kainos) [inline conf/constants.py]: could these be moved into a class?

File: ui_automation_tests/pages/apply_for_a_licence_page.py [hunk 2]
```diff
@@ -6,9 +6,9 @@ class ApplyForALicencePage(BasePage):
 
     NAME_OR_REFERENCE_INPUT_ID = "name"
     LICENCE_TYPE_PARTIAL_ID = "licence_type-"
-    MOD_APPLICATION_TYPE_PARTIAL_ID = "application_type-"
-    BUTTON_STANDARD_LICENCE = "input#application_type-standard_licence"
-    BUTTON_OPEN_LICENCE = "input#application_type-open_licence"
+    MOD_APPLICATION_TYPE_PARTIAL_ID = "case_type__sub_type-"
+    BUTTON_STANDARD_LICENCE = "input#case_type__sub_type-standard"
```
Comment (Mark-Spiers-kainos) [inline ui_automation_tests/pages/apply_for_a_licence_page.py]: can these be id's?

**Comment by jfaracik** [comment]
What does `self.application["case_type"]` return?

File: conf/constants.py (hunk 1 above)
Comment (andytaylorkainos) [reply conf/constants.py]: > What does `self.application["case_type"]` return?

case_type["sub_type"]

**Comment by andytaylorkainos** [comment]
> What does `self.application["case_type"]` return?

Screen Shot 2020-02-13 at 14 14 50

File: ui_automation_tests/pages/apply_for_a_licence_page.py (hunk 2 above)
Comment (andytaylorkainos) [reply ui_automation_tests/pages/apply_for_a_licence_page.py]: reverted this code

**Comment by andytaylorkainos** [comment]
> > What does `self.application["case_type"]` return?
> 
> Screen Shot 2020-02-13 at 14 14 50

The API now returns key-value pairs for each of the dictionaries (type, reference, sub_type)

**Pull request merged**